<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KDiskMark</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_All">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SEQ_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SEQ 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SEQ_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SEQ 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RND_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RND 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RND_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RND 2</string>
    </property>
   </widget>
   <widget class="QWidget" name="commentLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>781</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="commentLayout">
     <item>
      <widget class="QLabel" name="extraIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deviceModel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="readWidget" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>48</y>
      <width>222</width>
      <height>303</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="readLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_Read">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="readBar_SEQ_1">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="readBar_SEQ_2">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="readBar_RND_1">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="readBar_RND_2">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="writeWidget" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>48</y>
      <width>222</width>
      <height>303</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="writeLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_Write">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="writeBar_SEQ_1">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="writeBar_SEQ_2">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="writeBar_RND_1">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="writeBar_RND_2">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="mixWidget" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>48</y>
      <width>222</width>
      <height>303</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mixLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_Mix">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Mix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="mixBar_SEQ_1">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="mixBar_SEQ_2">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="mixBar_RND_1">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="mixBar_RND_2">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="targetLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>681</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="targetLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <widget class="QSpinBox" name="loopsCount">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_fileSize">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Dirs">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_ComparisonField">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>MB/s</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GB/s</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IOPS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Î¼s</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_MixRatio">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuInterval_Time">
     <property name="title">
      <string>Interval Time</string>
     </property>
     <addaction name="action0_sec"/>
     <addaction name="action1_sec"/>
     <addaction name="action3_sec"/>
     <addaction name="action5_sec"/>
     <addaction name="action10_sec"/>
     <addaction name="action30_sec"/>
     <addaction name="action1_min"/>
     <addaction name="action3_min"/>
     <addaction name="action5_min"/>
     <addaction name="action10_min"/>
    </widget>
    <addaction name="menuInterval_Time"/>
    <addaction name="actionQueues_Threads"/>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
    <addaction name="actionDefault"/>
    <addaction name="actionPeak_Performance"/>
    <addaction name="actionReal_World_Performance"/>
    <addaction name="actionDefault_Mix"/>
    <addaction name="actionPeak_Performance_Mix"/>
    <addaction name="actionReal_World_Performance_Mix"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuProfile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action0_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0 sec</string>
   </property>
  </action>
  <action name="action1_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 sec</string>
   </property>
  </action>
  <action name="action3_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 sec</string>
   </property>
  </action>
  <action name="action5_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 sec</string>
   </property>
  </action>
  <action name="action10_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 sec</string>
   </property>
  </action>
  <action name="action30_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>30 sec</string>
   </property>
  </action>
  <action name="action1_min">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 min</string>
   </property>
  </action>
  <action name="action3_min">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 min</string>
   </property>
  </action>
  <action name="action5_min">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 min</string>
   </property>
  </action>
  <action name="action10_min">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 min</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About KDiskMark</string>
   </property>
  </action>
  <action name="actionQueues_Threads">
   <property name="text">
    <string>Queues &amp;&amp; Threads</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionPeak_Performance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Peak Performance</string>
   </property>
  </action>
  <action name="actionReal_World_Performance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Real World Performance</string>
   </property>
  </action>
  <action name="actionDefault_Mix">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default [+Mix]</string>
   </property>
  </action>
  <action name="actionPeak_Performance_Mix">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Peak Performance [+Mix]</string>
   </property>
  </action>
  <action name="actionReal_World_Performance_Mix">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Real World Performance [+Mix]</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
